static/assets/js/team.js

```diff
@@ -0,0 +1,8 @@
+$(document).ready(function(){
+	var parent = $(".team-list");
+	var divs = parent.children();
+	while (divs.length) {
```

Comment:
not sure this works. wouldn't that just be an endless loop if `divs.length` is >0?